flagenv_test.go

```diff
@@ -0,0 +1,42 @@
+package flagenv_test
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/bitnami-labs/flagenv"
+)
+
+func TestFlagenv(t *testing.T) {
+	testCases := []struct {
+		set  bool
+		val  string
+		want string
+	}{
+		{false, "", "default"},
+		{true, "bar", "bar"},
+		{true, "", ""},
+	}
+	for i, tc := range testCases {
+		t.Run(fmt.Sprint(i), func(t *testing.T) {
+			defer os.Unsetenv("MY_TEST_FOO")
+
+			if tc.set {
+				os.Setenv("MY_TEST_FOO", tc.val)
+			} else {
```

Comment:
Would be nice to remove the empty `else`.

Reply:
I guess I trust gofmt too much; nice catch :-)